.github/workflows/test.yaml

```
@@ -0,0 +1,12 @@
+on: ["push", "pull_request"]
+
+name: Test
+
+jobs:
+  functional:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ApeWorX/github-action@fix/pyproject-toml-support
```

Revert this once merged and released
```suggestion
    - uses: ApeWorX/github-action@v2
```